### ISSUE TITLE
0.11/0.12 compatibility fix

### ISSUE THREAD

**Issue by yaroslavvb** [rafaljozefowicz/lm#2]
Haven't validated if accuracy is unaffected, but for the model to run on 0.12, tf.nn.rnn_cell.LSTMCell needs to be changed to:
`cell = tf.nn.rnn_cell.LSTMCell(hps.state_size, hps.emb_size, num_proj=hps.projected_size, state_is_tuple=False)
`

**Comment by ciprian-chelba** [comment]
Would you please clarify in what file? model_utils.py defines the LSTMcell used by language_model.py.

**Comment by yaroslavvb** [comment]
Oh, oops, I was actually looking a slightly different version of this code which used `cell = tf.nn.rnn_cell.LSTMCell`, haven't paid attention, looks like this version refactored that out
